app/views/foods/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<p style="color: green"><%= notice %></p>
+
+<h1 class="text-center w-full text-lg text-body-dark font-bold mb-10">Foods</h1>
+
+<div class="foods flex justify-center">
+  <table class="table-auto w-4/5 border-separate text-slate-500 overflow-auto">
+    <thead>
+      <tr>
+        <th class="bg-thead whitespace-nowrap p-3 text-sm font-semibold text-textHeading text-left">
+          Food
+        </th>
+        <th class="bg-thead whitespace-nowrap p-3 text-sm font-semibold text-textHeading text-left">
+          Measurement Unit
```

Comment:
## Full-fills Requirements

Your tables full-fill the project requirements for the food index page. Great Job!🙌